packages/@lightningjs/ui-components/src/components/ListItem/ListItemPicker.js

```diff
@@ -143,11 +145,17 @@ export default class ListItemPicker extends ListItem {
         }
       });
     }
-    this._Picker.patch({
+    const patchObject = {
       visible: !this._collapse,
       h: this.style.descriptionTextStyle.lineHeight,
-      w,
-      items: this.options.map(option => ({
+      w
+    };
+    if (
+      this._optionsChanged ||
+      this._previousMode !== this.mode ||
+      this._previousTone !== this.tone
```

Comment:
I'm not quite sure this new approach is correct. If the use patches in any style changes for the description, then this if statement will also get skipped. Yes, we need to patch a new set of items if the options array changes, but otherwise we probably should be updating the items directly to apply any style/theme updates (which would include changes from mode and tone).

Reply:
You're right, I definitely overcomplicated it! I was able to revert this whole block back to what it originally was and just patch in the selected index to resolve the original issue.